opm/autodiff/BlackoilTransportModel.hpp

```diff
@@ -158,6 +158,13 @@ namespace Opm {
 
 
 
+        std::vector<V>
+        computeFluidInPlace(const ReservoirState& x,
+                            const WellState& xw,
+                            const std::vector<int>& fipnum)
+        {
+            return asImpl().computeFluidInPlace(x, xw, fipnum);
+        }
```

Comment:
If you used a vector reference from the `getRegions` call in parser it returns by value, not reference.